x/crosschain/types/status.go

```diff
@@ -13,7 +13,8 @@ func (m *Status) AbortRefunded() {
 func (m *Status) UpdateStatusAndErrorMessages(newStatus CctxStatus, statusMsg, errorMsg string) {
 	m.UpdateStatus(newStatus, statusMsg)
 
-	if newStatus == CctxStatus_Aborted || newStatus == CctxStatus_Reverted || newStatus == CctxStatus_PendingRevert {
+	if errorMsg != "" &&
+		(newStatus == CctxStatus_Aborted || newStatus == CctxStatus_Reverted || newStatus == CctxStatus_PendingRevert) {
```

Comment:
can we move status check to a function w/ a  comment? It's not clear why only these statuses are eligible

Reply:
+1

Reply:
I think we can simply remove these, if an error is provided you update the new error message in any case

Reply:
https://github.com/zeta-chain/node/pull/3226/commits/c00f9644a73f30771a74adeaa15fd9a6ffbdef40